components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilder.java

```diff
@@ -74,8 +79,8 @@ public static EventBroker createEventBroker() throws EventBrokerConfigurationExc
      */
     private static OMElement loadConfigXML() throws EventBrokerConfigurationException {
 
-        String carbonHome = System.getProperty(ServerConstants.CARBON_HOME);
-        String path = carbonHome + File.separator + "repository" + File.separator + "conf" + File.separator + EventBrokerConstants.EB_CONF;
+        String carbonConfigHome = CarbonBaseUtils.getCarbonConfigDirPath();
+        String path = Paths.get(carbonConfigHome, EventBrokerConstants.EB_CONF).toString();
```

Comment:
Shall we use Paths.get(carbonConfigHome, EventBrokerConstants.EB_CONF).toFile()

then no need to create a File Object in line 86

Reply:
Done